src/xnnpack/prefetch.h

```diff
@@ -6,7 +6,29 @@
 #pragma once
 
 #ifdef _MSC_VER
-  #include <intrin.h>
+  #include <intrin.h>  
+#endif
+
+#ifdef __hexagon__
+  #include <hexagon_protos.h>
+
+  // Use fetchl2 at least several hundred cycles prior to using the data.
+  // The second parameter (lines) sets the number of lines of 128bits to bring into L2 cache.
```

Comment:
128 bits? Or 128 bytes?

Reply:
128 bytes. Sorry I corrected them. Also I added more description + fixed prefetch_linear function to call right intrinsic.

For linear case, I needed to prepare `info` having
0-7: height
8-15: width
16-31: stride
I set width = stride = 128bytes here.